packages/core/src/Provider/NetworkInfoProvider/types.ts

```diff
@@ -1,20 +1,6 @@
 import { Cardano, EraSummary, Provider } from '../..';
 
-export type ProtocolParametersRequiredByWallet = Required<
-  Pick<
-    Cardano.ProtocolParametersBabbage,
-    | 'coinsPerUtxoByte'
-    | 'maxTxSize'
-    | 'maxValueSize'
-    | 'stakeKeyDeposit'
-    | 'maxCollateralInputs'
-    | 'minFeeCoefficient'
-    | 'minFeeConstant'
-    | 'minPoolCost'
-    | 'poolDeposit'
-    | 'protocolVersion'
-  >
->;
+export type ProtocolParametersRequiredByWallet = Omit<Cardano.ProtocolParameters, 'costModels'>;
```

Comment:
The reason I omit `costModels` is that it is the only properly added from `Alonzo` era now that does not exist in Blockfrost API [epoch_param_content](epoch_param_content) section. It's needed for the [mapping](https://github.com/input-output-hk/cardano-js-sdk/pull/416/files#diff-88e0e444f82c6720c69afd174e9e6a88e9300309d629a851c8571a42926f76c8R26-R60) of protocol parameters since now we return non-nullable composed `ProtocolParameters` type from `currentWalletProtocolParameters` within `cardano-services` and `blockfrost`.

With the latest db-sync schema, there is one column called `cost_model_id` in `epoch_param` but again it's not clear how it could be mapped to `CostModels` type exposed by `ProtocolParametersAlonzo` in Ogmios schema
@rhyslbw

Reply:
It's ok as wallet is not using this parameter. We could probably further narrow down this type to only includue parameters that are actually required by wallet.

Reply:
Let's try to include these now, as they are relevant to wallet, even if ours isn't using it yet:
![image](https://user-images.githubusercontent.com/303881/189057935-5da0573f-67f8-4fd5-86d6-a86814bd3d7b.png)

> On the db-sync side with the latest schema, there is one column called cost_model_id in epoch_param but again it's not clear how it could be mapped to CostModels type exposed by ProtocolParametersAlonzo in Ogmios schema

https://github.com/input-output-hk/cardano-db-sync/blob/master/doc/schema.md#cost_model

We'll have to infer the Plutus version based on sort order of `block_id`

Reply:
What is the link of this swagger doc? 👀

Reply:
https://docs.blockfrost.io/

Reply:
ah didn't notice it's the blockfrost from the screenshot, thanks